HW1/P2/P2b.py

```diff
@@ -1,3 +1,25 @@
 from P2 import *
+from pyspark import SparkContext
+import random
 
-# Your code here
+# Set up partitions and context
+N = 2000
+sc = SparkContext("local", "P2b")
+
+# Partition a 10x10 cartesian grid of workers
+# Same as P2a, except the ordering is randomized
+rdd = sc.parallelize(sorted(range(N), key=lambda k: random.random()), 10)
```

Comment:
Looking at the "Partitioner" docs right now and I don't quite see how this would work unless I actually hard-coded positions based on my prior understanding of Mandelbrot functions, which seems a bit iffy to me.